src/tools/gophish_import.py

```diff
@@ -69,19 +69,19 @@ def load_landings(api, assessment):
                 break
             except Error as e:
                 if e.message == "Page name already in use":
-                    logging.warning(f"{e}. Finding with previously loaded page.")
+                    logging.warning("%s. Finding with previously loaded page.", e)
                     old_pages = api.pages.get()
                     for old_page in old_pages:
                         if old_page.name == new_page.name:
-                            logging.debug(f"Deleting Page with ID {old_page.id}")
+                            logging.debug("Deleting Page with ID %i", old_page.id)
                             api.pages.delete(old_page.id)
                             logging.info("Re-Loading new page.")
                 else:
                     logging.error(f"{e}\n")
                     raise
 
         # Returns Landing Page ID
-        logging.info(f"Landing Page {new_page.name} loaded.\n")
+        logging.info("Landing Page %s loaded.\n", new_page.name)
```

Comment:
I don't think the `\n` is needed here.

Reply:
See 825962be74cfc038bdf689b711d0b91860d5c0b5